tests/ui/documents/test_documents_api.py

```diff
@@ -139,6 +139,10 @@ def test_document_can_delete_with_loans(
 
 def test_document_person_resolve_exception(es_clear, db, document_data_ref):
     """Test document person resolve."""
+    document_data_ref['authors'] = [{
```

Comment:
better to  add this in the file `tests/data/data.json` document= `doc2`?

Reply:
No, because I like to trigger only on this test the exception.